app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentUpdateTest.kt

```diff
@@ -148,4 +150,28 @@ class ScheduleAppointmentUpdateTest {
             )
         )
   }
+
+  @Test
+  fun `when patient defaulter status is loaded and patient is defaulter, then schedule an automatic appointment`() {
```

Comment:
This test is testing when the patient is a defaulter then do something but the commit message is for when not now is clicked. Can we change the commit message to represent what's happening here?

Reply:
That's intentional. When we say defaulter in commit it doesn't describe the behaviour. But in tests, the naming indicates what's happening.